src/main/drivers/timer_common.c

```diff
@@ -28,14 +28,15 @@ uint8_t timerIndexByTag(ioTag_t ioTag)
 {
 #ifdef USE_TIMER_MGMT
     for (unsigned i = 0; i < MAX_TIMER_PINMAP_COUNT; i++) {
-        if (timerIOConfig(i)->ioTag == ioTag) {
+        if (timerIOConfig(i)->index >= 0 && timerIOConfig(i)->ioTag == ioTag) {
             return timerIOConfig(i)->index;
         }
     }
+    return -1;
```

Comment:
The return type is `uint8_t`, so this should be `0xff`, or the return type should be changed.